src/main/java/com/github/cameltooling/lsp/internal/Runner.java

```diff
@@ -39,10 +39,22 @@
 	private static final String PORT_PARAMETER = "--port=";
 	private static final String HOSTNAME_PARAMETER = "--hostname=";
 	private static final String CONTEXTPATH_PARAMETER = "--contextPath=";
+	static final String HELP_PARAMETER = "--help";
+	
+	static final String HELP_MESSAGE =
+			"`--help` to display this message\n\n"
+			+ "When launching without parameter, the Language Client can use Standard Input and Ouput to connect to the Camel Language Server.\n\n"
+			+ "To use a websocket connection, the parameter `--websocket` must be provided.\n"
+			+ "In this case, 3 other parameters are available:\n"
```

Comment:
I would replace it with "If you are using the websocket connection, 3 additional parameters are available:"